python/pyarrow/tests/test_dataset.py

```diff
@@ -100,7 +100,6 @@ def assert_dataset_fragment_convenience_methods(dataset):
 
 
 @pytest.fixture
-@pytest.mark.parquet
```

Comment:
Do we need to keep this? (it does use parquet)

Reply:
Had to remove marks from fixtures to avoid future error, see

> PytestRemovedIn9Warning: Marks applied to fixtures have no effect
> See docs: https://docs.pytest.org/en/stable/deprecations.html#applying-a-mark-to-a-fixture-function

If I checked correctly all functions that use these fixtures have parquet marks themselves.